harc_game_web/apps/wotd/views.py

```diff
@@ -24,7 +24,10 @@ def __verify(self, word, guess):
         - Removing leading and trailing whitespace.
         - Replacing any whitespace or repeated dashes with single dashes.
         """
-        return slugify(word.answer) == slugify(guess)
+        try:
+            return slugify(word.answer) == slugify(guess)
+        except AttributeError:
```

Comment:
dlaczego?

Reply:
Albowiem jak nie ma hasła na dany dzień, to nie ma do niego odpowiedzi...